backend/src/controllers/webhooks.controller.ts

```
@@ -0,0 +1,1453 @@
+
+export class WebhooksController {
+  private sanitizeInput(input: string): string {
+    if (!input || typeof input !== "string") {
+      throw new Error("Invalid input");
+    }
+
+    // Remove potentially dangerous characters and normalize
+    return input
+      .trim()
+      .replace(/[<>"'&]/g, "") // Remove HTML/script injection chars
+      .replace(/[\x00-\x1F\x7F]/g, "") // Remove control characters
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)